.github/workflows/test-pr.yaml

```diff
@@ -68,7 +74,7 @@ jobs:
       - name: Run Azimuth tests
         uses: azimuth-cloud/azimuth-config/.github/actions/test@devel
 
-      # Tear down the environment
+      # Tear down the environment
```

Comment:
I randomly found this when I was looking for something else, it's a weird non-space-space unicode char. I've fixed it with a nice normal space